### PR TITLE
Implement "click to start" functionality and some minor refactoring

### PR DESCRIPTION
**Pull request by kgalushchak**

### CODE REVIEW

File: course-project/src/GameStore.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {
   changeDirection,
   getEyesPosition,
   getFood,
-  getFoodPosition,
+  getFoodPosition, getInitialSnakePosition,
```
Comment (avilde) [inline course-project/src/GameStore.ts]: Should be in seperate line

File: course-project/src/GameStore.ts [hunk 2]
```diff
@@ -20,23 +20,21 @@ class GameStore {
   HEIGHT = 600;
```
Comment (avilde) [inline course-project/src/GameStore.ts]: Haven't asked before but why these properties are with capital letters? Class property naming convention is different.

File: course-project/src/components/StartButton.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {observer} from 'mobx-react-lite';
+import {useStore} from '../GameStore';
+
+const StartButton = observer(() => {
+  const {resetGame} = useStore('Game');
+
+  return <button onClick={() => resetGame()}>Click to start</button>; //TODO make it work
```
Comment (avilde) [inline course-project/src/components/StartButton.tsx]: What exactly doesn't work?

File: course-project/src/gameCalculations.ts [hunk 4]
```diff
@@ -4,6 +4,14 @@ import {food} from './food';
 const LEFT_EYE_INDENT = 0.25;
 const RIGHT_EYE_INDENT = 0.75;
 
+export const getInitialSnakePosition = (width: number, height: number, step: number): Snake => {
+  return [
+    {x: width/2, y: height/2},
```
Comment (avilde) [inline course-project/src/gameCalculations.ts]: space before and after division symbol

**Review by avilde — COMMENTED** [review]
There are many warnings, all of them need to be resolved:
Screenshot 2021-07-09 at 10 42 30

File: course-project/src/GameStore.ts (hunk 1 above)
Comment (kgalushchak) [reply course-project/src/GameStore.ts]: Moved into separate line

File: course-project/src/GameStore.ts (hunk 2 above)
Comment (kgalushchak) [reply course-project/src/GameStore.ts]: Yes, you are right, moreover, these values are not supposed to changes throughout the app, so moved them out of the class

File: course-project/src/components/StartButton.tsx (hunk 3 above)
Comment (kgalushchak) [reply course-project/src/components/StartButton.tsx]: Everything was already fixed, forgot to remove the TODO

File: course-project/src/gameCalculations.ts (hunk 4 above)
Comment (kgalushchak) [reply course-project/src/gameCalculations.ts]: Added missing space

**Pull request merged**